client/blocks/login/error-notice.jsx

```diff
@@ -69,11 +70,20 @@ class ErrorNotice extends Component {
 			return null;
 		}
 
-		return (
-			<Notice status={ 'is-error' } showDismiss={ false }>
-				{ error.message }
-			</Notice>
-		);
+		let errorElement = <Notice status={ 'is-error' } showDismiss={ false }>
+			{ error.message }
+		</Notice>;
```

Comment:
That's minor but I would suggest enclosing HTML fragments with parentheses (here and below):

```javascript
let errorElement = (
  <Notice status={ 'is-error' } showDismiss={ false }>
    { error.message }
  </Notice>
);
```

I find this syntax more readable. Actually I wonder if that's not a convention that we usually follow in Calypso.